src/testSimulator/School.java

```diff
@@ -0,0 +1,29 @@
+package testSimulator;
+
+import java.util.ArrayList;
+
+public class School {
+
+	private ArrayList<Classroom> classrooms;
+	private ArrayList<ArrayList<Answer>> answers;
```

Comment:
Great idea, bro - multidimensional array to store answers when you're using object-oriented programming. Don't even show me that again.